waspc/e2e-test/E2eTestDriver.hs

```diff
@@ -0,0 +1,23 @@
+import GoldenTest (GoldenTest)
```

Comment:
Nice, to have file like this!

Even if this file was named `Main`, it would be ok with me. `Driver` usually doesn't have actual tests in it, it is more of a logic for `driving` the tests. But ok.

As for your question about automatic answer -> I don't know the answer right now but I wouldn't worry about it too much, manually adding tests is not really a problem in practice, from what I observed. If it becomes we will figure it out.

Reply:
Sounds good. Yeah I may go back to Main until I can automatically discover them in this file. 👍🏻